README.md

```diff
@@ -31,10 +31,9 @@ Before calling the API methods, you need to create an instance of the `IncogniaA
 
 ```java
 // to use the BR region
-IncogniaAPI api = new IncogniaAPI("your-client-id", "your-client-secret", Region.BR);
-// to use the US region
-IncogniaAPI api = new IncogniaAPI("your-client-id", "your-client-secret", Region.US);
+IncogniaAPI api = new IncogniaAPI("your-client-id", "your-client-secret");
 ```
+Ideally you should use the instance of api as a singleton, so it can correctly generate tokens in the right interval.
```

Comment:
```suggestion
Ideally you should use the instance of IncogniaAPI as a singleton, so that it can properly handle token renewal.
```
What do you think?